fixieai/cli/agent/commands.py

```diff
@@ -497,36 +497,52 @@ def _watch_tunnel():
                         break
                     agent_api.update_agent(func_url=url)
                     console.print(
-                        f"[yellow]Tunnel URL was updated, now serving via {url}[/yellow]"
+                        f"[yellow]Tunnel URL was updated. Now serving via {url}[/yellow]"
                     )
 
             threading.Thread(target=_watch_tunnel, daemon=True).start()
         else:
             agent_api = _ensure_agent_updated(client, agent_id, config)
 
         console.print(
-            f"🦊 Agent [green]{agent_api.agent_id}[/] running locally on {host}:{port}, served via {agent_api.func_url}"
+            f"🦊 Agent [green]{agent_api.agent_id}[/] running locally on {host}:{port}, served via {agent_api.func_url}. To interact with this agent, open another terminal window, and run: `fixie console --agent {agent_api.agent_id}`.`"
```

Comment:
I think this is the type of messaging we should have more of – we should hold the dev's hand through every step of the process.

Reply:
Good call. I think a pretty slick UX would be to make a TUI that splits the console and shows logs on the right and automatically starts a new session with the agent on the left. But it quickly spirals out of control with UX design problems :)

Reply:
For sure! Asking someone to open a new terminal window feels silly.

Alternatively, maybe we want to get rid of `fixie console` and just push people towards using the webapp? Are there particular advantages/use-cases that `fixie console` offers that we can't do on the web?